app/src/main/java/com/dozingcatsoftware/bouncy/GL20Renderer.java

```diff
@@ -162,16 +162,14 @@ float world2glX(float x) {
         float scale = Math.max(cachedWidth, cachedHeight);
         float wx = fvManager.world2pixelX(x);
         float offset = 1 - cachedWidth / scale;
-        float glx = ((2 * wx) / scale - 1) + offset;
-        return glx;
+        return ((2 * wx) / scale - 1) + offset;
```

Comment:
I'd rather keep the separate variable here and for `gly` below. I think it's slightly more clear and makes debugging easier.